src/app/authentication/auth.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+declare var Msal: any;
+@Injectable()
+export class AuthService {
+    private applicationConfig: any = {
+        clientID: (window as any).ClientId,
+        graphScopes: ['openid', 'profile', 'User.ReadWrite', 'User.ReadBasic.All', 'Sites.ReadWrite.All',
```

Comment:
Its better we have some constants file for scopes from which we can use the scopes